packages/snap/src/index.ts

```diff
@@ -139,6 +145,8 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             },
           });
         }
+      } else { 
+        return;
```

Comment:
why did you add this?

Reply:
oh maybe add a little comment

Reply:
i can add a comment, but if we follow the if clause we see that we see that we return if user confirmation is false ( i.e if result === false )